src/harness/unittests/session.ts

```diff
@@ -175,6 +175,7 @@ namespace ts.server {
                 CommandNames.Configure,
                 CommandNames.Definition,
                 CommandNames.DefinitionFull,
+                CommandNames.DefinitionAndBoundSpan,

```

Comment:
I notice that the regular `Definition` command has a separate Full version.  Does the new command need one as well?

Reply:
Offline, I think we established that VS Code will probably not consume the new API.  If this API is just for VS, should it have "Full" in the name?  I think that's how we usually distinguish.

---
In reply to: [144675978](https://github.com/Microsoft/TypeScript/pull/19175#discussion_r144675978) [](ancestors = 144675978)

Reply:
Based on a discussion with @mhegazy, I have added the method for the simplified, hence, the "full" has been updated.